From 7368963f068760996ee9cde4744b962b85528706 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 Jan 2023 10:53:19 +0200
Subject: [PATCH 40/63] arch/arm64: Enable STM driver for S32CC platforms

STM7 is used for CAN timestamping and has to be enabled by default.

Issue: ALB-9261
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/Kconfig.platforms | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index fd703e3be86a..54bcb4012ff6 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -20,6 +20,7 @@ choice
 		select NXP_ERRATUM_ERR050543
 		select MAILBOX
 		select MULTIPLEXER
+		select NXP_GLOBAL_TIME_STM
 		select MUX_MMIO
 		select ARM_SMC_MBOX
 		select ARM_SCMI_PROTOCOL
-- 
2.25.1

